.dockerignore

```diff
@@ -0,0 +1,3 @@
+venv/**
+
+.env
```

Comment:
We do not seem to be using any `.env` file so I think it can be safely excluded from this file. What do you think?

Reply:
Yes, we can remove it if we are not using it. I will do that.

Reply:
Changed it,  please check.